static_files/genesis-generation-config/el/genesis-config.yaml.tmpl

```diff
@@ -1,12 +1,13 @@
 # NOTE: This does NOT have any relevance to the mnemonics & validator keys in the CL genesis!
 mnemonic: "stumble horn valley travel milk void screen bulk wink hood cup item glove setup wrong toward erase invite saddle this poverty basket index lab"
 el_premine:
-  "m/44'/60'/0'/0/0": 10000000ETH
-  "m/44'/60'/0'/0/1": 10000000ETH
-  "m/44'/60'/0'/0/2": 10000000ETH
-  "m/44'/60'/0'/0/3": 10000000ETH
-  "m/44'/60'/0'/0/4": 10000000ETH
-  "m/44'/60'/0'/0/5": 10000000ETH
+  "m/44'/60'/0'/0/0": 10000ETH
```

Comment:
Is there a reason why you reduced the amount of ETH here?

Reply:
oh mistake will correct